src/components/Navbar/Navbar.tsx

```diff
@@ -1,35 +1,71 @@
 import React, { FC, memo } from "react";
-import { Link as RouterLink } from "react-router-dom";
+import { useLocation } from "react-router-dom";
 import AppBar from "@material-ui/core/AppBar";
 import Toolbar from "@material-ui/core/Toolbar";
 import Typography from "@material-ui/core/Typography";
-import Link from "@material-ui/core/Link";
-import Button from "@material-ui/core/Button";
+import ExitToAppIcon from "@material-ui/icons/ExitToApp";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import Badge from "@material-ui/core/Badge";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import Tooltip from "@material-ui/core/Tooltip";
+import { RouterLocation } from "types/routeDef";
+import { ROUTE_LIST } from "routes/Routes.config";
 import { useStyles } from "./Navbar.styles";
 
-const Navbar: FC = () => {
+interface NavbarProps {
+  sideNavToggle: () => void;
+}
+const Navbar: FC<NavbarProps> = ({ sideNavToggle }) => {
   const classes = useStyles();
-  const accessToken = false;
+  const accessToken = true;
+  const location: RouterLocation = useLocation();
+  const pageTitle = ROUTE_LIST.find(route => route.path === location.pathname)
```

Comment:
make a `useMemo` to avoid traversing the `ROUTE_LIST` on every render.

Reply:
I have added the useMemo hook.